ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__ota-multi-stable-4.15-upgrade-from-stable-4.14.yaml

```diff
@@ -78,6 +78,21 @@ tests:
     - ref: cucushift-upgrade-healthcheck
     - ref: cucushift-ota-postupgrade
     workflow: cucushift-installer-rehearse-azure-ipi
+- as: aws-ipi-efa-pg-f28
+  cron: 32 15 5 * *
+  steps:
+    cluster_profile: aws-1-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      ENABLE_OTA_TEST: OCP-23799
```

Comment:
I did not see the preupgrade and postupgrade for this case, so I think we do not need to enable `ENABLE_OTA_TEST` here.

Reply:
Yes, this job is just used to make sure --to-latest work fine, so no pre and post upgrade steps are needed. but if we removed ENABLE_OTA_TEST then how do we know OCP-23799 is automated?

Reply:
We should fill automation field in polarion case, to map the ci job name.